### ISSUE TITLE
TypeError: Cannot read property 'unprefixed' of undefined

### ISSUE THREAD

**Issue by jens-duttke** [kristerkari/stylelint-high-performance-animation#24]
After updating PostCSS to version 8, the following error occurs in my code:
```
(node:1600) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'unprefixed' of undefined
    at P:\HexEd.it\src\scss\bootstrap.scss:132:3
    at P:\HexEd.it\src\scss\bootstrap.scss:134:5
    at P:\HexEd.it\node_modules\stylelint-high-performance-animation\index.js:218:36
    at P:\HexEd.it\node_modules\postcss-scss\node_modules\postcss\lib\container.js:190:18
    at P:\HexEd.it\node_modules\postcss-scss\node_modules\postcss\lib\container.js:135:18
    at Rule.each (P:\HexEd.it\node_modules\postcss-scss\node_modules\postcss\lib\container.js:101:16)
    at Rule.walk (P:\HexEd.it\node_modules\postcss-scss\node_modules\postcss\lib\container.js:131:17)
    at P:\HexEd.it\node_modules\postcss-scss\node_modules\postcss\lib\container.js:148:24
    at AtRule.each (P:\HexEd.it\node_modules\postcss-scss\node_modules\postcss\lib\container.js:101:16)
    at AtRule.walk (P:\HexEd.it\node_modules\postcss-scss\node_modules\postcss\lib\container.js:131:17)
    at AtRule.walkDecls (P:\HexEd.it\node_modules\postcss-scss\node_modules\postcss\lib\container.js:188:19)
    at P:\HexEd.it\node_modules\stylelint-high-performance-animation\index.js:217:23
```

I'm using:
postcss 8.0.5
stylelint 13.7.1
stylelint-high-performance-animation 1.5.1
NodeJS: 14.10.1

There are two issues:
1. In your plugin PostCSS is used without to specify the supported version as `peerDependency`.
2. `postcss.vendor` has been removed in PostCSS 8, see https://github.com/postcss/postcss/issues/1372
  Like described in this ticket the fix would be to copy the `unprefixed` function into your code (See https://github.com/postcss/postcss/blob/be01ca14ba92d02ed6b90941755aa9c7e7ad234b/lib/vendor.js)
  There are multiple places in your code using this function - replacing them all fixes the issue (I've tested that locally).

**Comment by SharakPL** [comment]
@jens-duttke could you make PR with the fix?

**Comment by jens-duttke** [comment]
@SharakPL Please see my pull request.

**Comment by SharakPL** [comment]
@jens-duttke Works fine :)

**Comment by kristerkari** [comment]
I'll release a new version now.

**Comment by kristerkari** [comment]
Thanks! Fixed in 1.5.2 https://github.com/kristerkari/stylelint-high-performance-animation/releases/tag/1.5.2

**Comment by RishabSurana** [comment]
if you are using **postcss** version 8, then make sure you also have the latest verison of **postcss-preset-env**, as you would not want to downgrade **postcss** due to security issues associated with version less than 8
